packages/core-data/src/actions.js

```diff
@@ -466,7 +466,7 @@ export const saveEntityRecord = (
 				// so the client just sends and receives objects.
 				const currentUser = select.getCurrentUser();
 				const currentUserId = currentUser ? currentUser.id : undefined;
-				const autosavePost = resolveSelect.getAutosave(
+				const autosavePost = await resolveSelect.getAutosave(
```

Comment:
How did we miss it? :o great catch!

Reply:
> The situation we'd need looks like: the post has edits for content, a previous autosave for title, and now we want to do a new autosave. And we want to autosave the full { content, title } tuple because both attributes are changed. But under what circumstances is it possible that the title changes are in autosave but not in local edits?

@jsnajdr Hm perhaps when if the autosave was created in one editing session, then the page was refreshed, and the new editor uses that autosave but without any local edits? On the other hand, if the `await` was missing for a few months and noone noticed, maybe we could remove that entirely 🤔  let's try that in a follow-up PR

Reply:
> Hm perhaps when if the autosave was created in one editing session, then the page was refreshed, and the new editor uses that autosave but without any local edits?

In that case, when I refresh the page, the editor tells me that there is an existing autosave and asks if I want to apply it? If I don't, it's discarded, if I do, the autosave changes become edits?

> if the `await` was missing for a few months and noone noticed any difference, maybe we could remove that entirely

It's also possible that people were actually losing data in a rare edge case, without ever having enough info to report an issue, and that's not a good thing. We should certainly investigate this further.